libpromises/mod_custom.c

```diff
@@ -325,7 +334,8 @@ static JsonElement *PromiseModule_Receive(PromiseModule *module, const Promise *
             for (size_t i = 0; i < length; ++i)
             {
                 logged_error |= PromiseModule_LogJson(
-                    JsonArrayGet(json_log_messages, i), pp, promise_log_level);
+                    JsonArrayGet(json_log_messages, i), pp,
+                    promise_log_level, changes_allowed);
```

Comment:
Reading these binpacked / Z shape lists of arguments is really annoying to me, but maybe just what you are used to.